web/docs/understand/data-model/type-system.md

```diff
@@ -6,35 +6,298 @@ sidebar_position: 0
 
 One [design goal](/docs/understand/architecture/design-goals) of VAST is
 being expressive enough to capture the semantics of the domain. This led us to
-develop a rich type system for structured security data, inspired by the data
-model of the [Zeek](https://zeek.org) network security monitor.
+develop a rich type system for structured security data, inspired by the
+[Zeek](https://zeek.org) network security monitor.
+
+## Terminology
+
+The diagram below illustrates the type system at a glance:
+
+![Type System - VAST](type-system-vast.excalidraw.svg)
+
+All type have an optional name. They coexist in a global namespace.
```

Comment:
`s/type/types` and I would move this to the alias section below, as naming only becomes relevant with them.